lib/dialogue-agent/entity-linking/entity-finder.js

```diff
@@ -96,4 +97,8 @@ function getBestEntityMatch(searchTerm, entityType, candidates) {
     return best;
 }
 
+function _removeParenthesis(str){
```

Comment:
By convention, only private methods and private fields of classes start with _. Functions should not start with _